stable/hl-composer/values.yaml

```diff
@@ -0,0 +1,145 @@
+# Default values for hl-composer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+persistence:
+  # By default we must set this to false to enable chart to pass tests
+  # However, in a proper deployment, you should set this to "true", so that the different components can share data
+  enabled: false
+  annotations: {}
+  capacity: 1Gi
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  storageClass: ""
+  accessMode: ReadWriteMany
+
+cli:
+  image:
+    repository: hyperledger/composer-cli
+    tag: 0.20.0
+    pullPolicy: IfNotPresent
+
+  secrets: {}
```

Comment:
I'm not sure this is the best way to handle these secrets.   I'd probably have a single `secret` file with each one as a key.  You could provide default values that would be loaded into the secret, and provide an override secretName to allow for a pre-existing secret containing the values.

Reply:
Hi Paul,

Thanks for the comments.

There are no "default" values for these, as the admin certificate and key will vary from deployment to deployment. Also, sometimes you want to mount only the certificate, but not the key, as is the case for the hlf-ord chart.

The blockchain network will also vary widely from deployment to deployment, and is also a file that is not easily written in ASCII/Unicode. Typically this file would be generated programatically with the Composer CLI  and then loaded by the Composer REST server server.

Finally, the HLF-Connection secret is also wildly varying depending on the architecture of the network (how many peers there are, what are their addresses, where are the orderers, etc.)

This is why I defined them in this "optional" way.